src/app/page.tsx

```diff
@@ -2,31 +2,53 @@ import About from "@/components/sections/about";
 import FAQ from "@/components/sections/faq";
 import Schedule from "@/components/sections/schedule";
 import { SparklesCore } from "@/components/sections/sparkles";
-
+import { TimerContainer } from '@/components/sections/TimerContainer';
 import Alt1 from "@/assets/logos/alt 1.svg";
+import { Button, MovingBorder } from "@/components/sections/moving-border";
+import { TypewriterEffectSmooth } from "@/components/sections/typewriter-effect";
+import Footer from "@/components/footer/footer";
 
 export default function Home() {
-  return (    
-    <main className="flex min-h-screen bg-black flex-col items-center justify-center p-24 relative">
-      {/* <h1 className="text-6xl font-bold">MCRStudentHack</h1> */}
-      <div className="w-full h-full absolute inset-0">
-        <SparklesCore
-          id="tsparticlesfullpage"
-          background="transparent"
-          minSize={0.6}
-          maxSize={1.4}
-          particleDensity={100}
-          className="w-full h-full"
-          particleColor="#FFFFFF"
-        />
-      </div>
-      <div className="flex min-h-screen flex-col items-center justify-center p-24 relative" >
-        <Alt1 className="transition ease-in-out delay-150 hover:scale-110 content-center" />
+  return (
+    <main className="flex min-h-screen bg-black flex-col items-center justify-center p-4 sm:p-24 relative" >
+      <div className="w-full h-full absolute inset-0"></div>
+        <div className="w-full h-full absolute inset-0">
+          <SparklesCore
+            id="tsparticlesfullpage"
+            background="transparent"
+            minSize={0.6}
+            maxSize={1.4}
+            particleDensity={100}
+            className="w-full h-full"
+            particleColor="#FFFFFF"
+          />
+        </div>
+        <div className="flex min-h-screen flex-col items-center justify-center p-4 sm:p-24 relative">
```

Comment:
Move elements to a file under @components/sections/hero.tsx or landing.tsx and not directly inject it here.


Additionally use fragments instead of div if possible